gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_2/00_add_event_organizations.yml

```diff
@@ -13,6 +13,14 @@ databaseChangeLog:
             columns:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.